src/vs/platform/terminal/common/terminal.ts

```diff
@@ -232,6 +233,8 @@ export interface IPtyService {
 	setTerminalLayoutInfo(args: ISetTerminalLayoutInfoArgs): Promise<void>;
 	getTerminalLayoutInfo(args: IGetTerminalLayoutInfoArgs): Promise<ITerminalsLayoutInfo | undefined>;
 	reduceConnectionGraceTime(): Promise<void>;
+	requestDetachInstance(workspaceId: string, instanceId: number): Promise<IProcessDetails | undefined>;
+	acceptDetachedInstance(requestId: number, process: number): Promise<IProcessDetails | undefined>;
```

Comment:
Including "reply" here would make the relationship with `requestDetachInstance` clearer:  `acceptDetachInstanceReply` or `acceptRequestDetachInstanceReply`